Confus/Confus/EventManager.h

```diff
@@ -4,21 +4,33 @@
 
 namespace Confus
 {
+    /// <summary> 
+    /// Event Manager
```

Comment:
Will do later, I only added those summary's for my report so they are not 100% complete yet etc.